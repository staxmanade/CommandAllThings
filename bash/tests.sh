
DIR="$( cd "$( dirname "$0" )" && pwd )"

# include the sample aliases
. $DIR/sampleProfile.sh

commands=( grunt gulp jake rake gradle make )

for i in "${commands[@]}"
do

	testDir=$(cd "$DIR/../samples/${i}Sample"; pwd)

	echo "*********************************"
	echo "*** $i"
	echo "*** Expected Testing Dir: $testDir"

	cd $testDir

	echo "*** Current Directory : $(pwd)"
	
    for j in "${commands[@]}"
    do
		if [ -e "package.json" ]; then
            npm install > /dev/null
		fi
		echo "*** Executing: $j"
	    eval $j
    done
done

