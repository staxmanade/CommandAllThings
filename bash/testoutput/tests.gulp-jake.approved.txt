[gulp] Using gulpfile REMOVED
[gulp] Starting 'test'...
Hello Gulp
[gulp] Finished 'test' after ##.# μs
[gulp] Starting 'default'...
[gulp] Finished 'default' after ##.# μs
